cdc/entry/schema_test_helper.go

```diff
@@ -60,7 +61,7 @@ func NewSchemaTestHelper(t *testing.T) *SchemaTestHelper {
 // DDL2Job executes the DDL stmt and returns the DDL job
 func (s *SchemaTestHelper) DDL2Job(ddl string) *timodel.Job {
 	s.tk.MustExec(ddl)
-	jobs, err := s.GetCurrentMeta().GetLastNHistoryDDLJobs(1)
+	jobs, err := tiddl.GetLastNHistoryDDLJobs(s.GetCurrentMeta(), 1)
```

Comment:
Seems more changes are introduced in this PR. Are they related to the IPv6 issue or they are fixed here simply because they violate the integration tests or some other stuffs due to some upgrades.

Reply:
because dependency is updated